### PR TITLE
Pf 500

### PR DESCRIPTION
**Pull request by owennicol**
#### What does this PR do? (please provide any background)
https://hxshortbreaks.atlassian.net/browse/FP-500
This sets `<strong />` tags within `<p>` tags to be non-capitalised
Also adds an `.no-arrow` class to buttons for when we don't want an arror
#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
Nope
#### Screenshots / Screencast
*Look - no arrow on the button!*
screen shot 2017-08-24 at 10 21 14

### CODE REVIEW

File: brands/thorpe-park/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holidayextras/brand-thorpepark",
-  "version": "3.1.4",
+  "version": "3.1.6",
```
Comment (jodiedoubleday) [inline brands/thorpe-park/package.json]: why have you jumped 2 versions?

File: brands/thorpe-park/src/less/type.less [hunk 2]
```diff
@@ -91,3 +91,7 @@ h1, h2, h3, h4, h5, h6,
 strong{
```
Comment (jodiedoubleday) [inline brands/thorpe-park/src/less/type.less]: pernickety can you add a space before {

File: brands/thorpe-park/src/less/type.less [hunk 3]
```diff
@@ -91,3 +91,7 @@ h1, h2, h3, h4, h5, h6,
...
   text-transform: uppercase;
 }
+
+p strong{
```
Comment (jodiedoubleday) [inline brands/thorpe-park/src/less/type.less]: and here

File: brands/thorpe-park/package.json (hunk 1 above)
Comment (owennicol) [inline brands/thorpe-park/package.json]: Because my diff said it was on version `3.1.5` already
screen shot 2017-08-24 at 11 01 28

Comment (owennicol) [inline brands/thorpe-park/package.json]: I have not 👋

**Pull request merged**